spark-connector/v3.3/spark/build.gradle.kts

```diff
@@ -135,5 +135,7 @@ tasks.test {
 }
 
 tasks.clean {
+  delete("derby.log")
```

Comment:
Do we also need to add them to `.gitgonre`?

Reply:
done